integrations/argocd/client.py

```diff
@@ -5,6 +5,8 @@
 from loguru import logger
 
 from port_ocean.utils import http_async_client
+import deprecation  # type: ignore
```

Comment:
There is a linting error thrown by mypy because the package does not have a strict type def

```
client.py:8: error: Skipping analyzing "deprecation": module is installed, but missing library stubs or py.typed marker  [import-untyped]
client.py:8: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
```

Reply:
lets stay with it for now, maybe we will add it to ocean framework in the future